common/djangoapps/util/password_policy_validators.py

```diff
@@ -42,12 +44,12 @@ def validate_password_length(value):
     message = _("Invalid Length ({0})")
     code = "length"
 
-    min_length = getattr(settings, 'PASSWORD_MIN_LENGTH', None)
-    max_length = getattr(settings, 'PASSWORD_MAX_LENGTH', None)
+    min_length = PASSWORD_MIN_LENGTH
```

Comment:
@schenedx I see. May be we need to override these settings for stage and prod to apply our change that we made in #15699